### PR TITLE
updated sortval to remove span tag & removed spurious </td>

### PR DESCRIPTION
**Pull request by jwgulley**
I have tested this fix on:
https://trade-remedies-caseworker-dev.london.cloudapps.digital/case/3fede0bb-6c8c-4fb4-889f-7da6435ae39a/files/

### CODE REVIEW

File: trade_remedies_caseworker/templates/cases/files.html [hunk 1]
```diff
@@ -40,7 +40,7 @@ <h3 class="heading-small">Showing for {{submission.name}}</h3>
 
 {% for document in file_list %}
     <tr>
-        <td class="nobreak" sortVal="{% format_date document.created_at '%Y-%m-%d' %}">
+        <td class="nobreak" sortVal="{{document.created_at}}">
```
Comment (markhigham) [inline trade_remedies_caseworker/templates/cases/files.html:43]: Shouldn't this be `{{ document.created_at|format_date:'%Y%m%d'}}`

**Review by markhigham — CHANGES_REQUESTED** [review]

File: trade_remedies_caseworker/templates/cases/files.html [hunk 2]
```diff
@@ -40,8 +40,8 @@ <h3 class="heading-small">Showing for {{submission.name}}</h3>
 
 {% for document in file_list %}
     <tr>
-        <td class="nobreak" sortVal="{% format_date document.created_at '%Y-%m-%d' %}">
-            {{ document.created_at|format_date:'%d %b %Y' }}
+        <td class="nobreak" sortVal={{ document.created_at|format_date_no_span:'%Y_%m_%d'}}  >
```
Comment (markhigham) [inline trade_remedies_caseworker/templates/cases/files.html:43]: I checked this. Not sure why the built in date doesn't work!! But this is a good solution.
You just need to surround the {{ with some quotes

`sortVal="{{ document.created_at|format_date_no_span:'%Y_%m_%d'}}"`

**Review by markhigham — CHANGES_REQUESTED** [review]

File: trade_remedies_caseworker/templates/cases/files.html (hunk 2 above)
Comment (jwgulley) [reply trade_remedies_caseworker/templates/cases/files.html]: Thanks for your kind feedback, Mark.  I've now added the quote marks, tested it and updated the PR.

**Review by markhigham — APPROVED** [review]
:octocat: TMLG

**Pull request merged**